src/Admin.php

```diff
@@ -114,7 +114,10 @@ public static function getProductShortestDeliveryTime(\WC_Product $product): str
 
     foreach ($product->get_children() as $variation) {
       $variation_term_id = get_post_meta($variation, '_lieferzeit', TRUE);
-      $variation_term_slug = get_term($variation_term_id)->slug;
+      $variation_term_slug = get_term($variation_term_id)?->slug;
+      if (!$variation_term_slug) {
+        continue;
+      }
```

Comment:
Cleaner:
```suggestion
      if (!$variation_term = get_term($variation_term_id)) {
        continue;
      }
      $variation_term_slug = $variation_term->slug;
```

Reply:
I don't like assigning stuff inside if statements at all. But thanks for the suggestion